src/openforms/js/components/form/content.js

```diff
@@ -25,7 +33,7 @@ const CUSTOM_CSS_CLASS = {
 
 const CONTENT_PRESENTATION = {
   ...PRESENTATION,
-  components: [SHOW_IN_EMAIL, SHOW_IN_PDF],
+  components: [SHOW_IN_SUMMARY(false), SHOW_IN_EMAIL, SHOW_IN_PDF],
```

Comment:
In other places we did `{...SHOW_IN_SUMMARY, defaultValue: false}`. For example here: https://github.com/open-formulieren/open-forms/blob/master/src/openforms/js/components/form/edit/tabs.js#L50

Maybe the above is a less invasive solution? Also in general we expect these components to be 'dicts' rather than functions, so maybe it's good to remain consistent?